### ISSUE TITLE
Docker images are way too big

### ISSUE THREAD

**Issue by abailly-iohk** [input-output-hk/mithril#528]
The mithril-client docker image weighs about 1.4GB which is about twice a similar haskell image, and we know Haskell is not very thin :) It's not clear to me why they are so large but we should be able to trim them down.

**Comment by jpraynaud** [comment]
@abailly, after verification, the images were very big before we merged this PR #525 (where we updated the base image)

Now they are `~120MB` for the Client: (same for the Signer/Aggregator)
```
REPOSITORY                               TAG            IMAGE ID       CREATED        SIZE
ghcr.io/input-output-hk/mithril-client   latest         d11d65da6a96   24 hours ago   120MB
ghcr.io/input-output-hk/mithril-client   main-46f719c   ffa21583c62c   4 days ago     120MB
ghcr.io/input-output-hk/mithril-client   main-2de1b9b   74ad4167725a   5 days ago     1.33GB
ghcr.io/input-output-hk/mithril-client   main-367fb76   b277cafdd161   13 days ago    1.33GB
```

**Comment by abailly** [comment]
Thanks a lot 🙏 !
